src/views/ResultsPage/ResultsPage.jsx

```diff
@@ -623,65 +623,64 @@ const ResultsPage = () => {
 					content="Find an NCI-supported clinical trial - Search results"
 				/>
 			</Helmet>
-			<article className="results-page">
-				{error.length && error.filter((err) => err.fieldName === 'zip') ? (
-					<>{renderInvalidZip()}</>
-				) : (
-					<>
-						{checkIfInvalidPage() ? (
-							<></>
-						) : isLoading || !searchCriteriaObject ? (
-							<div className="loader__pageheader"></div>
-						) : (
-							<>
-								<h1>Clinical Trials Search Results</h1>
 
-								<ResultsPageHeader
-									resultsCount={trialResults.total}
-									pageNum={currentPage}
-									onModifySearchClick={handleRefineSearch}
-									onStartOverClick={handleStartOver}
+			{error.length && error.filter((err) => err.fieldName === 'zip') ? (
+				<article className="results-page">{renderInvalidZip()}</article>
+			) : (
+				<article className="results-page">
+					{checkIfInvalidPage() ? (
+						<></>
+					) : isLoading || !searchCriteriaObject ? (
+						<div className="loader__pageheader"></div>
+					) : (
+						<>
+							<h1>Clinical Trials Search Results</h1>
+
+							<ResultsPageHeader
+								resultsCount={trialResults.total}
+								pageNum={currentPage}
+								onModifySearchClick={handleRefineSearch}
+								onStartOverClick={handleStartOver}
+								searchCriteriaObject={searchCriteriaObject}
+								isLoading={isLoading}
+								trialResults={trialResults}
+							/>
+						</>
+					)}
+					<div className="results-page__content">
+						{checkIfInvalidPage() ? <> </> : <>{renderControls()}</>}
```

Comment:
Looks like we can clean this up a little bit and simplify the render logic

Reply:
Yup. Can move the container `<article className="results-page">` up and have the conditional rendering occur inside of the container for one.